src/main/scala/com/linkedin/feathr/offline/join/workflow/AnchoredFeatureJoinStep.scala

```diff
@@ -126,7 +128,7 @@ private[offline] class AnchoredFeatureJoinStep(
     val anchoredFeaturesThisStage = featureNames.filter(allAnchoredFeatures.contains).map(allAnchoredFeatures).distinct
     val anchoredDFThisStage = anchorDFMap.filterKeys(anchoredFeaturesThisStage.toSet)
     // map feature name to its transformed dataframe and the join key of the dataframe
-    val featureToDFAndJoinKeys = transformFeatures(anchoredDFThisStage, anchoredFeatureNamesThisStage, bloomFilter)
+    val featureToDFAndJoinKeys = transformFeatures(anchoredDFThisStage, anchoredFeatureNamesThisStage, bloomFilter, None, mvelContext)
```

Comment:
Wouldn't it be cleaner if this variable were named expressionContext or something everywhere, to keep closer to what the type is called?